engine/src/main/java/com/agiletec/aps/system/common/AbstractGenericCacheWrapper.java

```diff
@@ -64,13 +65,15 @@ protected void insertAndCleanCache(Cache cache, Map<String, O> objects) {
     protected void insertAndCleanCache(Cache cache, Map<String, O> objects, String codesCacheKey, String cacheKeyPrefix) {
```

Comment:
why we cannot put a check to not null and return an error? don't you think we are complicating too much the base common class (is it safe ?) to handle a case of a caller that wants just to clear the cache for a key? ( ... and should use another method ... )

Reply:
Restored previous method